app/lib/ui/components/switches/component_switches.dart

```diff
@@ -23,7 +24,7 @@ class _ComponentSwitchesState extends State<ComponentSwitches> {
   Widget build(BuildContext context) {
     return Scaffold(
         appBar: AppBar(
-          title: Text('Switches'),
+          title: MainAppBar('Switches'),
```

Comment:
étrange, ne devrait pas être:
return Scaffold(
       appBar: MainAppBar('Radio Buttons'),
        body:

au lieu de 

return Scaffold(
        appBar: AppBar(
          title: MainAppBar('Switches'),

Reply:
It was a mistake